Form.css

```diff
@@ -0,0 +1,24 @@
+.Form {
+  width: 50%;
```

Comment:
Flex element width can be set using `flex-basis` (and `flex-grow`/`flex-shrink`).